atmosphere_unit_service/service/config.py

```diff
@@ -0,0 +1,74 @@
+from dataclasses import dataclass
+
+@dataclass
+class AmpqProps:
+    addr: str
+    port: str
+    username: str
+    password: str
+    queue_name: str
+
+@dataclass
+class SelfProps:
+    grpc_addr: str
+    grpc_port: str
+
+    event_queue_props: AmpqProps
+
+@dataclass
+class ScenaristProps:
+    event_queue_props: AmpqProps
+
+class Config:
+
+    def __init__(self) -> None:
+        self.self._props = SelfProps(
```

Comment:
Сломанная конструкция

```suggestion
        self.self_props = SelfProps(
```